tests/integration/telemetry/stats/prometheus/wasm/wasm_modules/header_injector/Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: docker-push docker-build build clean
+
+VERSION_LIST := 1 2
```

Comment:
I may miss something, when will this image build?
should we move this to sample folder?

Reply:
At this time, as mentioned in https://github.com/istio/istio/issues/38877, many images including the sample folder are created and pushed in ad-hoc way, and this one is so as well.

I have a plan to make an another PR to move this to Standard Build Process like `echo` server/client, but it could be changed by the discussion in https://github.com/istio/istio/issues/38877.

Reply:
make sense to me

Reply:
I think all the go stuff should be part of standard process, but not this. This is expensive to build due to c++ and bazel, and we do not even have bazel in our build tools

Reply:
To clarify - current state is good and I think should remain this way.

Reply:
I see. will keep it as it. Thanks for the comment.

Reply:
correct me if I'm wrong.
`header_injector` is built manual in local, and push to `gcr.io/istio-testing`?
other go projects(echo etc.) are built by `make docker.pushx`, which will run in CI flow?

Reply:
That is the long term plan, right now some go projects are still manual (see https://github.com/istio/istio/issues/38877)